### PR TITLE
Fixed accent and scandinavian letters

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search/scroll/ScrollQuery.java [hunk 1]
```diff
@@ -117,9 +117,7 @@ public static class ScrollBuilder {
         private String ttl;
 
         public ScrollQuery build() {
-            return nonNull(openSearchUri)
-                ? new ScrollQuery(ttl, firstResponse).withOpenSearchUri(openSearchUri)
-                : new ScrollQuery(ttl, firstResponse);
+              return new ScrollQuery(ttl, firstResponse).withOpenSearchUri(openSearchUri);
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search/scroll/ScrollQuery.java:120]: OpensearchUri is mandatory, should never be empty

File: search-commons/src/main/java/no/unit/nva/search/scroll/ScrollQuery.java [hunk 2]
```diff
@@ -107,7 +107,7 @@ private boolean shouldStopRecursion(Integer level, SwsResponse previousResponse)
     }
 
     private String scrollRequestToString(SearchScrollRequest request) {
-        return attempt(() -> toXContent(request, XContentType.JSON, true).utf8ToString())
+        return attempt(() -> toXContent(request, XContentType.JSON, ToXContent.EMPTY_PARAMS, true).utf8ToString())
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search/scroll/ScrollQuery.java:110]: deprecated

File: search-commons/src/test/java/no/unit/nva/search/BeforeAfterSuiteListener.java
Comment (StigNorland) [inline search-commons/src/test/java/no/unit/nva/search/BeforeAfterSuiteListener.java:1]: improve test speed, by reusing instance of opensearch through all tests

**Review by joachimjorgensen — APPROVED** [review]
LGTM! Looks very good and clean 👍 By how much was the testing time improved?

**Comment by StigNorland** [comment]
Surprisingly little in the build job, but on local machine, a minute.

**Pull request merged**